### DISCUSSION TITLE
How do I Keymap the Enter Key?

### DISCUSSION THREAD

**Discussion by Tookre** [rsjaffe/MIDI2LR#635] [Q&A] [answered]
Hello, im using MIDI2LR with the behringer x-touch mini.
I want to Keymap the "Enter" key, to press "done" after i used a brush, radial filter, crop etc.
In the moment when i want to add the key in the MIDI2LR settings by pressing the enter key the dialog just close.
Typing in enter also won't work.
Any Tipps?

![image](https://user-images.githubusercontent.com/76123305/155716674-939919f7-659f-41ac-9ad0-a6b553127f6a.png)

**Comment by rsjaffe** [comment, marked as answer]
See https://github.com/rsjaffe/MIDI2LR/wiki/Plugin-Options-Dialog#keyboard-shortcuts. Use `return` Instead.
